### PR TITLE
Add env var to disable the neumeric token that is used to string CC o…

### PR DESCRIPTION
**Pull request by hjm-aws**
…ps togather.

### CODE REVIEW

File: torch_xla/csrc/token_handler.cpp [hunk 1]
```diff
@@ -30,8 +31,19 @@ xla::XlaOp SliceOneToken(xla::XlaOp input) {
...
 }  // namespace
 
+#ifndef PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE
+#define PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE false
+#endif
+
 xla::XlaOp TokenHandler::GetInput(xla::XlaOp input,
                                   const xla::Shape* input_shape) {
+  static bool disable_numeric_token =
+      xla::sys_util::GetEnvBool("PT_XLA_DISABLE_NUMERIC_CC_TOKEN",
```
Comment (JackCaoG) [inline torch_xla/csrc/token_handler.cpp:41]: we usually don't prefix our env var with `PT_XLA`, maybe `DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE` directly?

File: torch_xla/csrc/token_handler.cpp [hunk 2]
```diff
@@ -30,8 +31,19 @@ xla::XlaOp SliceOneToken(xla::XlaOp input) {
...
+                                PT_XLA_DISABLE_NUMERIC_CC_TOKEN_DEFAULT_VALUE);
```
Comment (JackCaoG) [inline torch_xla/csrc/token_handler.cpp:42]: I think it is OK to hard code `false` directly here. We generally don't use `ifdef` in pt/xla.

**Comment by JackCaoG** [comment]
rebase should solve the build issue.

File: torch_xla/csrc/token_handler.cpp (hunk 2 above)
Comment (hjm-aws) [reply torch_xla/csrc/token_handler.cpp:42]: OK, we'll append the following to the end of `torch_xla/__init__.py`:
```
_set_missing_env('DISABLE_NUMERIC_CC_TOKEN', 'true')
```

File: torch_xla/csrc/token_handler.cpp (hunk 1 above)
Comment (hjm-aws) [reply torch_xla/csrc/token_handler.cpp]: Done

**Review by JackCaoG — APPROVED** [review]

**Comment by JackCaoG** [comment]
Thanks, I will merge the pr one all test passed.

**Pull request merged**